src/Microsoft.AspNetCore.Http/Internal/RoutingAddress.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Microsoft.AspNetCore.Http.Internal
+{
+    public class RoutingAddress
```

Comment:
It would be dope to avoid the name routing if possible - or make this internal internal . This isn't the routing package and isn't integrated with routing.

Reply:
I thought it was supposed to be BindingAddress?

Reply:
Yeah oops.